src/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -322,19 +323,19 @@ public virtual object Clone()
         }
 
         // V1 API.
-        public static Delegate CreateDelegate(Type type, object target, string method)
+        public static Delegate? CreateDelegate(Type type, object target, string method)
         {
             return CreateDelegate(type, target, method, false, true);
         }
 
         // V1 API.
-        public static Delegate CreateDelegate(Type type, object target, string method, bool ignoreCase)
+        public static Delegate? CreateDelegate(Type type, object target, string method, bool ignoreCase)
```

Comment:
In what situation can this return null?

Reply:
Because it it is set to null on 369. I'll change to return null and use non nullable.

Reply:
> Because it it is set to null on 369

This is why I've been stressing we need to first annotate the public surface area the way it should be annotated, and only then fix up the implementation. :wink: